src/patternfly/components/Popover/examples/Popover.md

```diff
@@ -131,35 +138,159 @@ import './Popover.css'
   {{/popover-content}}
 {{/popover}}
 ```
+### Default alert
+
+```hbs
+{{#> popover popover--modifier="pf-m-top pf-m-default" popover--attribute='aria-labelledby="popover-default-alert-header" aria-describedby="popover-default-alert-body"' popover--IsAlert="true" popover--IsDefaultAlert="true"}}
+  {{#> popover-content}}
+    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Close"'}}
+      <i class="fas fa-times" aria-hidden="true"></i>
+    {{/button}}
+    {{#> popover-header}}
+      {{#> popover-title popover-title--IsIcon="true" popover-title--modifier="pf-m-default" popover-title--attribute=(concat 'id="' popover-title--id '-title"')}}
+        Default alert title
+      {{/popover-title}}
+    {{/popover-header}}
+    {{#> popover-body popover-body--attribute='id="popover-default-alert-body"'}}
+      Popovers are triggered by click rather than hover. Click again to close.
+    {{/popover-body}}
+    {{#> popover-footer}}
+      Popover footer
+    {{/popover-footer}}
+  {{/popover-content}}
+{{/popover}}
+```
+### Success alert
+
```

Comment:
Nit - can you remove the space between the header and the start of the hbs block backticks?

Reply:
Fixed